client/src/components/ProductCard.tsx

```diff
@@ -143,8 +145,34 @@ function ProductCard ({ product, bookmark, log }: Props) {
     }
   };
 
+  const handleShareClick: React.MouseEventHandler<HTMLDivElement> = async () => {
+    const projectURL = 'https://oriori.fly.dev/home';
+    try{
+        if(navigator.share){
+          await navigator.share({
+            title: `${product.product_name}`,
+            text: `${product.id}:${product.product_name}`,
+            url: `${projectURL}#${product.id}`,
+          })
+        } else {
+          const shareURL = `${projectURL}#${product.id}`;
+            await navigator.clipboard.writeText(shareURL);
+            console.log("💌",product.id, ":", product.product_name,":", shareURL);
+        }
+        setHasShared(true);
+        setTimeout(() => {
+          setHasShared(false);
+        }, 3000);
+      } catch (error){
+        console.error("💩",error);
+    }
+  } 
+  
+
+
   return (
-    <div className='product__card'>
+    <div className='product__card' id={`${product.id}`}>
+      <a href={`#${product.id}`}>{product.id}</a>
```

Comment:
I figured this `<a>` tag to help with development but I just wanted to leave a note to make sure we don't forget to remove it before approving the PR. ;)

Reply:
@SirrorsMoore1975 If you wouldn't mind, could you go ahead and remove line 175 now so I can test and make sure that everything functions correctly without it?